non-critical-infra/modules/mailserver/default.nix

```
@@ -41,6 +47,26 @@
     path = "${config.mailserver.dkimKeyDirectory}/nixos.org.mail.key";
   };
 
+  sops.secrets."nixos.org.${arc.selector}.key" = {
+    format = "binary";
+    owner = "rspamd";
+    group = "rpsamd";
+    mode = "0400";
+    # rspamadm dkim_keygen --selector arc-2025 --domain nixos.org --type rsa --bits 2048
+    sopsFile = ../../secrets/nixos.org-${arc.selector}-private-key.umbriel;
+    path = "/var/lib/rspamd/arc/nixos.org.${arc.selector}.key";
+  };
+
+  services.rspamd = {
+    overrides."arc.conf".text = ''
+      domain {
+        nixos.org {
+          selector = "${arc.selector}";
```

nit: unnecessary string interpolation?


```suggestion
          selector = arc.selector;
```

---

No this is a string. It's needed.